### ISSUE TITLE
Unnecessary dependencies

### ISSUE THREAD

**Issue by emrekirmizi** [ModelWriter/WP3#22]
###### \* org.eclipse.mylyn.wikitext.markdown.core

-> We have EcoreDropAdapter class to connect IDragAndDropService on startup.
In this class, we are using MarkdownParser to parse markdown text, and it creates dependency on plugins above.
###### \* org.eclipse.mylyn.wikitext.core

-> In this project, we have MarkdownToReqIfDocumentBuilder which extends org.eclipse.mylyn.wikitext.core.DocumentBuilder
###### \* org.eclipse.mylyn.wikitext.markdown.core.

-> In this project, we have CreateReqifSpecFromMarkdownHandler class. If this handler is triggered, we are creating MarkdownParser instance, setting its document builder as MarkdownToReqIfDocumentBuilder, and running its parser. 
# In conclusion,

-> Maybe we should create our own parser or any other alternative.
